docs/readmes/howtos/troubleshooting/datapath_connectivity.md

```diff
@@ -196,6 +196,34 @@ on UE or the server (on SGi side of the network) while debugging the issue.
     the trace output in a github issue.
 15. If this document does not help to debug the issue, please post output of
     all steps in new github issue.
+16. Check whether GTP port `2152` is listening or not by using `sudo netstat -a|grep 2152`.
+    If it is not listening, check the `OVS` status using `sudo ovs-vsctl show`.
+17. If the below error is observed by running `sudo ovs-vsctl show`
+
+```Port gtp0
```

Comment:
`Port gtp0` needs to go onto the line below otherwise it isn't shown in the docs.